### DISCUSSION TITLE
All the adapters' charts create and release

### DISCUSSION THREAD

**Discussion by Aisuko** [meshery/meshery#1800] [Ideas] [answered]
As the https://github.com/layer5io/meshery/issues/1782 , we need to create the adapters' charts and release them to https://artifacthub.io/

- [x] OSM    #1802 

- [x] NSM #1900

- [x] Kuma   #1854

**Comment by Aisuko** [comment]
@leecalcote Should we add the chart into every adapters project and maintain their own chart. I mean as we can see we push all charts into the layer5io/meshery repo

**Reply by kushthedude** [reply]
+1 to this, the Adapter chart should reside in their respective repositories rather than being in with Meshery.

**Comment by leecalcote** [comment]
// @ramrodo

**Comment by leecalcote** [comment, marked as answer]
Adapters remain centrally sourced.
